presentation/common/src/presentation-common/PresentationRpcInterface.ts

```diff
@@ -137,7 +137,7 @@ export class PresentationRpcInterface extends RpcInterface {
   public static readonly interfaceName = "PresentationRpcInterface"; // eslint-disable-line @typescript-eslint/naming-convention
 
   /** The semantic version of the interface. */
-  public static interfaceVersion = "2.6.3";
+  public static interfaceVersion = "2.6.4";
```

Comment:
Adding a new RPC method should increase the minor version.

Reply:
Fixed